modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -26,7 +26,8 @@ def show
 
         def submit2122
           shared_form_validation('2122')
-          poa_code = parse_and_validate_poa_code('2122')
+          validate_registration_number!
+          poa_code = parse_and_validate_poa_code(FORM_NUMBER)
```

Comment:
we use the constant here but not a few lines above. Wondering if we should align these and maybe create one for `INDIVIDUAL` and `ORGANIZATION` (or Rep?) to make more sense? I think we may be dependent on `FORM_NUMBER` elsewhere, but should we align these values throughout the controller where possible? 

thoughts on this as well @dellerbie ?

Reply:
could tack the above changes on to a ticket specific to cleaning these up if we so desire.